tslint.json

```diff
@@ -13,6 +13,7 @@
       "allow-pascal-case",
       "allow-leading-underscore"
     ],
+    "no-submodule-imports": false,
```

Comment:
If we are going to use the setting then better set it to true and exclude `vscode`. 
i. e. `no-submodule-imports: [true, "vscode"]`

Reply:
I'm used to have this enabled from my work with `Angular` and `rxjs` but given that here we don't use `webpack` and we don't do any kind of `tree-shaking` I can live with it 😉

Reply:
Fine.